main_windows.go

```diff
@@ -0,0 +1,36 @@
+//go:build windows
+// +build windows
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/docker/pkg/reexec"
+
+	"github.com/containernetworking/cni/pkg/version"
+	bv "github.com/containernetworking/plugins/pkg/utils/buildversion"
+	hostlocal "github.com/containernetworking/plugins/plugins/ipam/host-local"
+	"github.com/containernetworking/plugins/plugins/main/windows/win-bridge"
+    "github.com/containernetworking/plugins/plugins/main/windows/win-overlay"
```

Comment:
nit: whitespace. Are you using an editor that supports autoformatting? If not, you can use `go fmt` to clean it up for you.
```suggestion
	"github.com/containernetworking/plugins/plugins/main/windows/win-overlay"
```

Reply:
@brandond thanks for catching that! I think my editor is using spaces by default. The formatting should be corrected now.